### PR TITLE
Spelling fixes and improved wording & legibility

### PR DESCRIPTION
**Pull request by Arxari**
Went over all pages except for guides, hardware & legal.
Made some grammar spelling fixes and changed some minor wording for better legibility

Made a change at docs/troubleshooting/shockosc.md which I am unsure of (but hoping it's correct)

 _#Never gonna give you up_

### CODE REVIEW

File: docs/diy/compile.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Compiling from source
 
 !!! info "Still brewing!"
-    This article heavily under development, expect very frequent changes
+    This article heavily under development; expect very frequent changes
```
Comment (LostQuasar) [inline docs/diy/compile.md:4]:
```suggestion
    This article heavily under development; expect frequent changes.
```
Redundant very and punctuation.

File: docs/diy/assembling.md [hunk 2]
```diff
@@ -6,12 +6,12 @@ This guide mainly focuses on the parts listed in the [Hardware Buy guide](../qui
 + ESP-32
 + 433 MHz transmitter
 + Soldering station
-+ Hookup wire (Optional depending if the pins line up)
++ Hookup wire (optional, depending on whether the pins line up)
```
Comment (LostQuasar) [inline docs/diy/assembling.md:9]:
```suggestion
+ Hookup wire (Optional, depending on whether the pins line up)
```
Taking out this capitalization seems awkward the rest looks good

File: docs/troubleshooting/shockosc.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ Go ahead and click reset config in the VRChat action menu. This should reload yo
 or
 
 If it doesn't, navigate to `%userprofile%\appdata\locallow\VRChat\VRChat` and delete `OSC` folder.
-Then switch avatar again.
+Then reload your avatar again.
```
Comment (LostQuasar) [inline docs/troubleshooting/shockosc.md:24]: This looks good just need @LucHeart or @Kyobinoyo to confirm.

**Review by LostQuasar — CHANGES_REQUESTED** [review]

File: docs/diy/assembling.md (hunk 2 above)
Comment (Arxari) [reply docs/diy/assembling.md:9]: Well optional is an adjective. Adjectives and nouns are usually lowercase unless they mark the beginning of a stance or are a proper noun, none of which are the case here.

Aka I see no reason why not to change it.

Comment (LostQuasar) [reply docs/diy/assembling.md:9]: IMO being linguisticly correct is not the goal here, it's understanding. To me `(Optional) is a serpate thought and `, depending on whether the pins line up)`  explains that secondary thought. Again I'm just nitpicking at this point.

Comment (Arxari) [reply docs/diy/assembling.md:9]: I guess that makes sense

**Review by LucHeart — APPROVED** [review]

**Review by LucHeart — APPROVED** [review]

**Pull request merged**